### PR TITLE
Fix elo and prices for analyics

### PR DESCRIPTION
**Pull request by dekkerglen**
Fixes an issue with populate analytics where it would not fetch elo and prices correctly

### CODE REVIEW

File: serverjs/cubefn.js [hunk 1]
```diff
@@ -225,6 +225,16 @@ function maybeCards(cube, carddb) {
   return maybe.map((card) => ({ ...card, details: carddb.cardFromId(card.cardID) }));
 }
 
+async function getCardElo(cardname) {
+  const rating = await CardRating.findOne({ name: { $regex: new RegExp(cardname, 'i') } });
```
Comment (phulin) [inline serverjs/cubefn.js:229]: This won't use the Mongo index and will thus be slow. Probably fine for where the function is being used now, but we're going to want to fix this in the future.

**Comment by phulin** [comment]
Also, CI failing.

File: serverjs/cubefn.js (hunk 1 above)
Comment (dekkerglen) [reply serverjs/cubefn.js:229]: This had reasonable perf when I ran it, and the size of that table grows extremely slow so I think it's ok

Comment (phulin) [reply serverjs/cubefn.js:229]: Yep, I'm just worried we'll accidentally let usage of this utility function sneak in to actual routes that get called much more frequently than populate_analytics.

**Pull request merged**